firebase-sessions/api.txt

```diff
@@ -3,16 +3,50 @@ package com.google.firebase.sessions {
 
   public final class FirebaseSessions {
     method @NonNull public static com.google.firebase.sessions.FirebaseSessions getInstance();
-    method @NonNull public static com.google.firebase.sessions.FirebaseSessions getInstance(@NonNull com.google.firebase.FirebaseApp app);
-    method public void register(@NonNull com.google.firebase.sessions.api.SessionSubscriber subscriber);
+    method @Deprecated @NonNull public static com.google.firebase.sessions.FirebaseSessions getInstance(@NonNull com.google.firebase.FirebaseApp app);
```

Comment:
Do you think we can just remove these instance and getInstance methods? Now there is no useful public api in this class anymore. Maybe even make FirebaseSessions internal. And only the stuff in the api package be public?

Reply:
I like it!